src/get-started/codelab.md

```diff
@@ -56,7 +56,7 @@ The animated GIF shows how the app works at the completion of part 1.
   <h4 class="no_toc">What you’ll learn in part 1</h4>
 
   * How to write a Flutter app that looks natural on iOS, Android,
-    and the web
+    Windows, and the web
```

Comment:
@RedBrogdon any particular reason this should be only windows? I'd be happy to mention all six platforms here.

Reply:
Also reworded this section to make it more generic